frontend/compute-system/src/App.tsx

```diff
@@ -22,7 +22,11 @@ function App() {
             path={routes.computationCockpit.path}
             component={ComputationCockpit}
           />
-          <Route exact path={routes.appDetails.path} component={AppDetails} />
+          <Route
+            exact
+            path={routes.appDetails.path}
+            render={(props) => <AppDetails {...props} />}
```

Comment:
Did you change it because you couldn't get path? It worked for me

Reply:
Path was set in the browser, but when I wanted to get it from code then it was `undefined`